src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -157,6 +167,86 @@ private void InitFamilyInstance(Autodesk.Revit.DB.FamilySymbol fs, XYZ pos)
             ElementBinder.SetElementForTrace(InternalElement);
         }
 
+        private void InitFamilyInstance(FamilySymbol fs, Reference reference, Line pos)
+        {
+            //Phase 1 - Check to see if the object exists and should be rebound
+            var oldFam =
+                ElementBinder.GetElementFromTrace<Autodesk.Revit.DB.FamilyInstance>(Document);
+
+            //There was a point, rebind to that, and adjust its position
+            if (oldFam != null)
+            {
+                InternalSetFamilyInstance(oldFam);
```

Comment:
Also we need to check if the oldFam is hosted on the same reference? If not, you may want to recreate a new family instance, else this may not be attached to the new face.